nixos/roles/router/bind/gocept.net.zone.static

```diff
@@ -9,7 +9,7 @@ ns2                     CNAME   ns2.gocept.com.
 ; generic services
 ; these aliases must match values in location.pp (if applicable)
 mail                    AAAA    2a02:248:101:62::19
-                        MX      10 mail
+                        MX      10 mail.flyingcircus.io.
```

Comment:
This change updates the MX record for `mail.gocept.net.` instead of the top-level MX record for `gocept.net.`. I presume this is probably not what you intended?

Reply:
(Does `mail.gocept.net` itself need an MX record? Can we remove that record from the zone file?)